molecule/default/tasks/openshift_adm_prune_deployments.yml

```diff
@@ -0,0 +1,103 @@
+- name: Prune deployments
+  block:
+  - set_fact:
+      dc_name: "hello"
+      prune_namespace: prune
+
+  - name: Ensure namespace
+    kubernetes.core.k8s:
+      kind: Namespace
+      name: '{{ prune_namespace }}'
+
+  - name: create deployment config
+    kubernetes.core.k8s:
```

Comment:
What about using `community.okd.k8s` instead to be consistent with the collection?

Reply:
Done. Thanks